### ISSUE TITLE
Do not tranfer selected not work ....

### ISSUE THREAD

**Issue by woshicsa2** [finndev/PokeBuddy#750]
config pidfey Do not tranfer :

![image](https://cloud.githubusercontent.com/assets/20841654/17560580/25b2d580-5f12-11e6-860f-a35df896f6ab.png)

but log :
[P - 16:48:29] [Navigation] Walking from: 37.78729, -122.40757 to: 37.78695, -122.40724 with a distance of: 29 meters
[P - 16:48:30] [Recycling Items] Recycling: 2 items
[P - 16:48:30] [Recycling Items] Recycled: ItemPokeBall x 6
[P - 16:48:31] [Transferring Pokemons] Transferring Pokemon away: Pidgey (CP: 410)!
[P - 16:48:31] [!] Access token expired. Reauthenticating...
[P - 16:48:32] [Recycling Items] Recycled: ItemPotion x 1
##  [P - 16:48:33] [Transferring Pokemons] Pidgey has been transferred
[P - 16:48:34] [Farm Pokestops] Spinning Pokestop: 'Number 57 Station San Francisco'
[P - 16:48:34] [Farm Pokestops] Success -> Received XP: 50

**Comment by curtiola** [comment]
Yeah its a known issue, I wish there was a sticky thread showing all the current bugs and plans for the future, but sadly there isn't
